niap-cc/Permissions/Tester/app/src/main/AndroidManifest.xml

```diff
@@ -26,7 +25,11 @@
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:debuggable="true"
+        android:testOnly="true"
+        android:process="system"
+        android:sharedUserId="android.uid.system"
```

Comment:
I think we should expect the considerable 'worst case' for platform signature application.
'sharedUserId' is removed from sdk29, so this line would not work, so I'll remove it.
give me your thought.

Reply:
While sharedUserId has been deprecated, it is still supported by the platform. I agree that a platform signed application could join this sharedUserId, but I think since the intent of the tester is to verify permissions behave as expected, we may still want to leave this line removed to ensure we're testing the permissions as implemented as opposed to any potential capabilities granted to an app running under the system sharedUserId.